pkg/internal/util/webhook/webhook.go

```diff
@@ -36,6 +42,24 @@ func IsDNS1123Label(s string) bool {
 	return dns1123LabelRegex.MatchString(s)
 }
 
+type LogLevelSetter interface {
+	SetLogLevel(int)
+}
+
+func SetDefaultLogLevel(ctx context.Context, c client.Client, spec LogLevelSetter) error {
+	kubeCarrier := &operatorv1alpha1.KubeCarrier{}
+	kubeCarrier.Name = constants.KubeCarrierDefaultName
+	objKey, err := client.ObjectKeyFromObject(kubeCarrier)
+	if err != nil {
+		return err
+	}
+	if err := c.Get(ctx, objKey, kubeCarrier); err != nil {
```

Comment:
```suggestion
	kubeCarrier := &operatorv1alpha1.KubeCarrier{}
	if err := c.Get(ctx, types.NamespaceName{Name: constants.KubeCarrierDefaultName}, kubeCarrier); err != nil {
```

simplify this, no need for those extra lines

Reply:
done